content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,113 @@
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual machine installation promoted to Beta
+
+With many users needing to expand their service mesh to include Virtual Machines from existing systems, managing the
+complexity of installation is paramount to user success. As such, Virtual Machine support has continued to be a high
+priority for the Istio community. Since Istio 1.6, the community has made major improvements each release for VM
+support. As of Istio 1.9, [Virtual Machine Installation](/docs/setup/install/virtual-machine/) has been promoted to Beta.
+
+All of the supporting documents for this feature have been updated to make it as easy as possible for you to add Virtual
+Machines to your service mesh:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+Similar to changes made for Virtual Machines, the Istio community has revamped and improved our
+[multicluster](/docs/setup/install/multicluster/) installation guides:
+
+* [Before you begin](/docs/setup/install/multicluster/before-you-begin/)
+* [Install Primary-Remote](/docs/setup/install/multicluster/primary-remote/)
+* [Install Primary-Remote on different networks](/docs/setup/install/multicluster/primary-remote_multi-network/)
+* [Verify installation](/docs/setup/install/multicluster/verify/)
+
+## `service-apis` promoted to Alpha
+
+Support for [Kubernetes `service-apis`](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's `Gateway` and `VirtualService`. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases. To try them out, check out the [getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
+These APIs are still evolving, so feedback is welcome!
```

Comment:
Why does a user care about this? Take a look at Istio theme docs if you like to add why a user should care about  this support.

Reply:
From a user's prospective I can think of a few reasons why:

1. Software agnostic APIs.
2. Given #1, it also allows a user to migrate towards Istio if they've already adopted K8s serice-apis
3. The networking APIs are deemed stable and will not be promoted to "v1". Over time they will be adopted. But it's not clear to me what the migration plans will be.

I know the networking team has talked about this quite often in recent meetings. It's still a bit too early to tell how Istio will be affected as the CRDs are still in active development and I'm not sure how much we should talk about the future for a feature that will see a lot of active development over the next year.

Reply:
I think the why should be allow moving from one service mesh easily to another service mesh who also adopted the k8s service API.

Reply:
How about

"We are eager to evolve these CRDs in partnership with the Kubernetes community in upcoming
releases to help unify and up-level Ingress capabilities across ecosystem."